python/ray/tune/logger.py

```diff
@@ -1,12 +1,15 @@
 import csv
 import json
 import logging
-import os
-import yaml
 import numbers
+import os
+from typing import TYPE_CHECKING, Callable, Dict, List, Optional, Type, Union
+
 import numpy as np
 
 import ray.cloudpickle as cloudpickle
+import yaml
```

Comment:
nit - we usually group ray imports / non-ray imports together (so `import yaml` would go before L10)